webapp/src/App.tsx

```diff
@@ -66,15 +52,13 @@ function App({ hubPort, autoReconnect }: AppProps) {
     return (
         <div>
             <Header
-                hubState={hubState}
                 isHubStateDialogOpen={isHubStateDialogOpen}
                 onHubStateDialogOpen={() => setIsHubStateDialogOpen(true)}
             />
             <div className="wrap">
                 <div className="left-frame" id="gap">
                     <div className="sidebar-buttons">
                         <MenuLeft
-                            selectedMode={hubState.daphbot_mode}
```

Comment:
mmmm... not sure if it's worth adding a dependency to useHubState  for a single item.  No need to change; just a thought.